missioncontrol/settings.py

```diff
@@ -311,6 +323,8 @@
 FIREFOX_VERSION_URL = 'https://product-details.mozilla.org/1.0/firefox_versions.json'
 FIREFOX_VERSION_CACHE_TIMEOUT = 300
 
+FIREFOX_EXPERIMENTS_URL = 'https://normandy.cdn.mozilla.net/api/v1/recipe/signed/?enabled=true'
```

Comment:
At least in experiments viewer the only ones we're interested in have an extra filter:
`https://normandy.cdn.mozilla.net/api/v1/recipe/signed/?enabled=true&latest_revision__action=3`